### DISCUSSION TITLE
Register issue

### DISCUSSION THREAD

**Discussion by samrudd** [JulianPrieber/littlelink-custom#135] [Issues]
When my users register I get a random error screen but if they refresh and go back it has inserted the info. 
![D2B4108E-67F0-410D-B6F7-93D38E247445](https://user-images.githubusercontent.com/45354394/185573724-3f1a82a3-a7f1-4607-bd73-e90c60dde1ab.jpeg)

**Comment by JulianPrieber** [comment]
If this error happens consistently, your best bet is probably to not use our SMTP server.

For me, everything is working fine, so it might have been a temporary thing.

The best solution for you is probably to use your own SMTP server, if you have one available.
Alternatively, if it is just for now, you can turn REGISTER_AUTH in your config from verified to auth. This allows users to still register but skip the email verification process.

**Reply by richard7ac** [reply]
Hi Julian - I got a similar issue with SMTP server on version 2 installed last week and had hoped it would go away with version 3 which I just installed but it has not. It does not seem to be intermittent.

stream_socket_enable_crypto(): SSL operation failed with code 1. OpenSSL Error messages: error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed

```
    public function startTLS()
    {
        // STREAM_CRYPTO_METHOD_TLS_CLIENT only allow tls1.0 connections (some php versions)
        // To support modern tls we allow explicit tls1.0, tls1.1, tls1.2
        // Ssl3 and older are not allowed because they are vulnerable
        // @TODO make tls arguments configurable
        return stream_socket_enable_crypto($this->stream, true, STREAM_CRYPTO_METHOD_TLSv1_0_CLIENT | 
              STREAM_CRYPTO_METHOD_TLSv1_1_CLIENT | STREAM_CRYPTO_METHOD_TLSv1_2_CLIENT);
    }


```
